spec/advanced_search_spec.rb

```diff
@@ -17,9 +15,7 @@
         expect(resp.size).to be >= 50
         expect(resp.size).to be <= 200
       end
-      it 'colour OR color photography', pending: 'fixme' do
-        # the following is busted due to Solr edismax bug that sets mm=1 if it encounters a OR
-        # https://issues.apache.org/jira/browse/SOLR-2649
+      it 'colour OR color photography' do
```

Comment:
So one question I have here now, is that this test isn't now testing what the SearchWorks query is for `OR`. Should we modify the test to do that instead?

Reply:
👍

Reply:
https://github.com/sul-dlss/sw_index_tests/pull/213